packages/phase2cli/.env.default

```diff
@@ -1,5 +1,4 @@
 # Firebase.
-FIREBASE_FIRESTORE_DATABASE_URL="YOUR-FIREBASE-FIRESTORE-DATABASE-URL-HERE"
```

Comment:
Why do you think it is right to remove it? Currently, we use the Firestore Database to store all the information. We can remove the Storage because we are on S3, but I don't think it is correct to remove the Firestore DB.

Reply:
Firestore doesn't use URL as an identifier anymore: https://github.com/firebase/firebase-js-sdk/pull/3650#issuecomment-681138610

Reply:
So, basically as long as we use the default naming inferred from project identifier we do not need explicit urls. That's cool!